docs/GettingStarted/configuration/ExporterWebhook.md

```diff
@@ -90,7 +90,7 @@ For the Header X-Pelorus-Event: `deploytime`
 | `app`       | `string` | Monitored application name |
 | `image_sha` | `string` | Image SHA used for deployment. Must be prefixed with the `sha256:` followed by 64 characters of small letters and numbers |
 | `namespace` | `string` | OpenShift namespace to which application was deployed |
-| `timestamp` | `int`    | EPOCH timestamp representing event occurrence. Allowed format: `10 digit int`|
+| `timestamp` | `int`    | EPOCH timestamp representing event occurrence. Must be not older then 30min from now. Allowed format: `10 digit int`|
```

Comment:
"then" here should be "than"?

I think we can rewrite this (I am afraid user can miss the "not" and understand it wrong)

what about `Maximum of 30 minutes old data from now`